src/providers/palm/complete.ts

```diff
@@ -0,0 +1,75 @@
+import { Params } from "../../types/requestBody";
+import { PalmCompleteResponse } from "../../types/responseBody";
+import { ChatCompletionResponse, ErrorResponse, ProviderConfig } from "../types";
+
+// TODOS: this configuration does not enforce the maximum token limit for the input parameter. If you want to enforce this, you might need to add a custom validation function or a max property to the ParameterConfig interface, and then use it in the input configuration. However, this might be complex because the token count is not a simple length check, but depends on the specific tokenization method used by the model.
+
+export const PalmCompleteConfig: ProviderConfig = {
+    model: {
+        param: "model",
+        required: true,
+        default: "model/text-bison-001",
+    },
+    prompt: {
+        param: "prompt",
+        default: "",
+        transform: (params: Params) => {
+            const { prompt: text } = params
+            const prompt = {
+                text,
+            }
+            return prompt
+        }
+    },
+    temperature: {
+        param: "temperature",
+        default: 1,
+        min: 0,
+        max: 1,
+    },
+    top_p: {
+        param: "topP",
+        default: 1,
+        min: 0,
+        max: 1,
+    },
+    top_k: {
+        param: "topK",
+        default: 1,
+        min: 0,
+        max: 1,
+    },
+    n: {
+        param: "candidateCount",
```

Comment:
candidate count is in snake case as per PALM docs.
https://developers.generativeai.google/tutorials/curl_quickstart

Reply:
Closing as per this https://github.com/Portkey-AI/Rubeus/pull/30/files#r1374132451